docs/applications/deluge.md

```diff
@@ -0,0 +1,60 @@
+# Deluge
+
+[Deluge](http://deluge-torrent.org/) is a lightweight, Free Software, cross-platform BitTorrent client.
+
+* Full Encryption
+* WebUI
+* Plugin System
+* Much more...
+
+
+[![deluge](https://avatars2.githubusercontent.com/u/6733935?v=3&s=200)](http://deluge-torrent.org/)
+
+
+## Usage
+
+Set `deluge_enabled: true` in your `roles/deluge/main.yml` file.
+
+Deluge's web interface can be found at http://ansible_nas_host_or_ip:32030
+
+Upon first viewing you will be prompted for a password. The default is `deluge` It is recommended that you change this password in the prefernces menu.
+
+**Please note that having multiple downloaders watching the same watchfolder, problems may occur.**
+
+## Parameters
```

Comment:
I'm not sure of the relevance of this section - the switches shown are what would be passed to docker, but obviously the whole point of Ansible-NAS is that people don't need to worry about poking Docker manually from the shell.

Reply:
the watch folder note was from some similar documentation that I was reading, but I can't find it again to cite, so I can re move that bit if needed. I don't mess much with watchfolders so I haven't ever tested it. 

the ##Parameters section is detailed information on the docker container for those who may want to tweak ansible-nas on their own. In my experience I have had to go and research each of the containers I use with ansible-nas though github or dockerhub and I figured it would be nice to have some of the more technical info on hand to the user. Alternatively I think we could add a blip like "For more technically detailed information on the container used by ansible-nas please visit (link.to.github) and add logo links to the individual projects. I think that ansible-nas can be a fantastic learning tool for some as well.

Reply:
I was referring to the parameters section specifically - the commands C&P'd from the linuxserver readme included `-p`, `-e` etc options, which are relevant to spinning up docker from the shell, not ansible. There are also options included that aren't exposed via the task/parameters, so it doesn't make sense to have documentation for them.